docker-compose.yml

```diff
@@ -19,5 +19,8 @@ services:
       - "8022:8022"
       - "2222:2222"
     volumes:
-      - ./:/usr/src/app
-    command: --log DEBUG
+      # keeps operation data & uploaded files between runs
+      - caldera-data:/usr/src/app/data
```

Comment:
Is there a reason why we're only storing the `data` directory for Caldera and not all files like before? When testing this locally, I had an issue because my `conf/local.yml` file was not persisting between docker compose runs, causing issues with loading existing data. Other edge cases that come to mind include plugin-specific data that would be excluded here, in particular the `atomic` plugin, which clones the Atomic Red Team repo on first time plugin load to convert the atomic tests into caldera abilities.